lib/buffer.js

```diff
@@ -104,6 +104,11 @@ let poolSize, poolOffset, allocPool;
 // do not own the ArrayBuffer allocator.  Zero fill is always on in that case.
 const zeroFill = bindingZeroFill || [0];
 
+const KnownBinaryEncoding = {
```

Comment:
This could use a more generic name, especially since ASCII isn't binary and the `'binary'` and `'latin1'` encodings are missing from this group (intentionally).

Reply:
Went with a more specific `EncodingsToUseWithBufferValues` ¯\\_(ツ)_/¯